hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->And(vixl::wzr, W(i.s1), MSKTOP(0xff), SetFlags);
```

Comment:
lpathy added `MSKTOP` originally. I'd recommend just adding a new function. While your at it, maybe we can rename these function from ALL-CAPS-MACRO-STYLE to something more verbose and camelCased. `Mask32` and `Mask8` maybe? What do you think?